Overview.bs

```diff
@@ -623,13 +626,479 @@ The algorithm:
 
 ### Patch Map Table: Format 2 ### {#patch-map-format-2}
 
-<!-- TODO add algorithm explaining how to union the two tables -->
-Coming soon.
+<!-- TODO: clean up unreferenced definitions -->
 
-<!-- 
-TODO:
-- how does this interact with woff2
--->
+<dfn>Format 2 Patch Map</dfn> encoding:
+
+<table>
+  <tr>
+    <th>Type</th><th>Name</th><th>Description</th>
+  </tr>
+  <tr>
+    <td>uint8</td>
+    <td><dfn for="Format 2 Patch Map">format</dfn></td>
+    <td>Set to 2, identifies this as format 2.</td>
+  </tr>
+  <tr>
+    <td>uint32</td>
+    <td><dfn for="Format 2 Patch Map">reserved</dfn></td>
+    <td>Not used, set to 0.</td>
+  </tr>
+  <tr>
+    <td>uint32</td>
+    <td><dfn for="Format 2 Patch Map">id</dfn>[4]</td>
+    <td>Unique ID used to identify patches that are compatible with this font.</td>
+  </tr>
+  <tr>
+    <td>uint8</td>
+    <td><dfn for="Format 2 Patch Map">defaultPatchEncoding</dfn></td>
+    <td>
+      Specifies the format of the patches linked to by uriTemplate (unless overridden by an entry). Uses the ID numbers from the
+      [[#font-patch-formats-summary]] table.
+    </td>
+  </tr>
+  <tr>
+    <td>uint16</td> <!-- TODO: uint32 to match format 1? -->
+    <td><dfn for="Format 2 Patch Map">entryCount</dfn></td>
+    <td>Number of entries encoded in this table.</td>
+  </tr>
+  <tr>
+    <td>Offset32</td>
+    <td><dfn for="Format 2 Patch Map">entries</dfn></td>
+    <td>Offset to a [=Mapping Entries=] sub table. Offset is from the start of this table.</td>
+  </tr>
+  <tr>
+    <td>uint16</td>
+    <td><dfn for="Format 2 Patch Map">uriTemplateLength</dfn></td>
+    <td>
+      Length of the uriTemplate string.
+    </td>
+  </tr>
+  <tr>
+    <td>uint8</td>
+    <td><dfn for="Format 2 Patch Map">uriTemplate</dfn>[uriTemplateLength]</td>
+    <td>
+      A [[!UTF-8]] encoded string. Contains a [[#uri-templates]] which is used to produce URIs associated with each entry.
+    </td>
+  </tr>
+</table>
+
+<dfn>Mapping Entries</dfn> encoding:
+
+<table>
+  <tr>
+    <th>Type</th><th>Name</th><th>Description</th>
+  </tr>
+  <tr>
+    <td>[=Mapping Entry=]</td>
+    <td><dfn for="Mapping Entries">entries</dfn>&lsqb;[=Format 2 Patch Map/entryCount=]&rsqb;</td>
+    <td>List of entries. Each entry has a variable length, which is determined following [$Interpret Format 2 Patch Map Entry$].</td>
+  </tr>
+</table>
+
+<dfn>Mapping Entry</dfn> encoding:
+
+<table>
+  <tr>
+    <th>Type</th><th>Name</th><th>Description</th>
+  </tr>
+  <tr>
+    <td>uint8</td>
+    <td><dfn for="Mapping Entry">format</dfn></td>
```

Comment:
Why not "flags" instead of "format"?

Reply:
Intention is to match the terminology used in GPOS ValueFormat which works similarly, but yeah this is probably a bit ambiguous with other uses of format numbers. How about formatFlags?

Reply:
formatFlags would work.